scripts/pylib/twister/twisterlib/harness.py

```diff
@@ -509,6 +489,116 @@ def _parse_report_file(self, report):
             self.instance.reason = 'No tests collected'
 
 
+class Robotframework(TwisterLibraryHarness):
```

Comment:
Should this name remain was `Robotframework` or rather use pascal case `RobotFramework`?

Reply:
Added to all files that were added in these commits